docs/using-cli.md

```diff
@@ -58,6 +46,37 @@ Note also that on Windows you need to use double quotes instead of single quotes
 
 Execute `jupytext --help` to access the full documentation.
 
+### Execute notebook cells
+
+For convenience, when creating a notebook from text you can execute it:
+
+```bash
+jupytext --set-kernel - notebook.md             # create a YAML header with kernel metadata matching the current python executable
+jupytext --set-formats md:myst notebook.md      # create a YAML header with an explicit jupytext format
+jupytext --to notebook --execute notebook.md    # convert notebook.md to an .ipynb file and run it
+```
+
+If you wanted to convert a collection of Markdown files to paired notebooks, and execute them in the current Python environment, you could run:
+```bash
+jupytext --set-formats ipynb,md --execute *.md
+```
+
+Please note: if any notebook cell errors, execution will terminate and `jupytext` will not save the notebook. This can cause headaches as the details of any error would be encoded in the notebook that isn't saved. But there's a way: `jupyter nbconvert` has a mode which will still save a notebook if a cell errors, producing something akin to what would happen if you ran all cells manually in Jupyter's notebook UI.
```

Comment:
I agree that having the notebook fail is an unpleasant experience, yet failing on error is also `nbconvert`'s standard, so I'd rather introduce the next paragraph as something like _advanced notebook execution_. What do you think?